### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by rpiaggio**
Updates to latest versions of `lucuma-core`, `lucuma-schemas` and `lucuma-ui`.

Some code adaptations were required.

This PR subsumes https://github.com/gemini-hlsw/observe/pull/657

### CODE REVIEW

File: modules/new_model/shared/src/main/scala/observe/model/Conditions.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+package observe.model
+
+import cats.Eq
+import cats.derived.*
+import observe.model.enums.*
+
+final case class Conditions(
+  cc: CloudCover,
+  iq: ImageQuality,
```
Comment (cquiroz) [inline modules/new_model/shared/src/main/scala/observe/model/Conditions.scala:12]: We have new classes (not enumerated) classes for conditions like `CloudExtinction`. Will we add those later on?

File: modules/new_model/shared/src/main/scala/observe/model/enums/SystemName.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.model.enums
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.eq.*
+import observe.model.ParamName
+
+enum SystemName(val system: String) derives Eq:
```
Comment (cquiroz) [inline modules/new_model/shared/src/main/scala/observe/model/enums/SystemName.scala:11]: Why not `derives Enumerated`?

File: modules/web/src/main/scala/observe/ui/Icons.scala [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+package observe.ui
+
+import react.fa.FAIcon
+import react.fa.FontAwesomeIcon
+import react.fa.IconLibrary
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.*
+
+object Icons:
```
Comment (cquiroz) [inline modules/web/src/main/scala/observe/ui/Icons.scala:13]: I wonder is some of these could go to `lucuma-ui` Maybe it just make things complicated?

File: modules/new_model/shared/src/main/scala/observe/model/Conditions.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/new_model/shared/src/main/scala/observe/model/Conditions.scala:12]: Yes, we should do that.

File: modules/new_model/shared/src/main/scala/observe/model/enums/SystemName.scala (hunk 2 above)
Comment (rpiaggio) [reply modules/new_model/shared/src/main/scala/observe/model/enums/SystemName.scala:11]: Good idea, I'll implement and remove the parser.

File: modules/web/src/main/scala/observe/ui/Icons.scala (hunk 3 above)
Comment (rpiaggio) [reply modules/web/src/main/scala/observe/ui/Icons.scala:13]: IDK what's best here, whether to have everything in one place or each system with their own references for the icons they use.

**Pull request merged**